src/util/usfm_to_json.js

```diff
@@ -207,7 +207,8 @@ module.exports = {
 };
 
 function fileName(file) {
-    return file.substr(file.lastIndexOf("/") + 1);
+    // return file.substr(file.lastIndexOf("/") + 1);
+    return (file.match("/") ? file.substr(file.lastIndexOf("/") + 1) : file.substr(file.lastIndexOf("\\") + 1));
```

Comment:
Is there a reason why we aren't using: https://nodejs.org/docs/latest-v10.x/api/path.html?

Reply:
Updated the changes